src/test/java/plugins/LdapPluginTest.java

```diff
@@ -73,6 +90,24 @@ public void login_ok() {
     }
 
     /**
+     * Scenario: Login with ldap uid and password
+     * Given I have a docker fixture "ldap" that allows anonymous binding
+     * And Jenkins is using ldap as security realm
+     * When I login with user "jenkins" and password "root" without manager credentials
+     * Then I will be successfully logged on as user "jenkins"
```

Comment:
Note, these are just a relic from BDD days and should not be added to new junit tests. I guess I should just go delete these comments everywhere not to confuse people.